gradient_utils/metrics.py

```diff
@@ -21,20 +21,26 @@ def _get_env_var_or_raise(*env_vars):
     return rv
 
 
-def _get_experiment_id():
+def _get_object_id():
     if os.getenv('PAPERSPACE_EXPERIMENT_ID'):
         return os.getenv('PAPERSPACE_EXPERIMENT_ID')
     hostname = os.getenv('HOSTNAME', "")
     try:
-        experiment_id = hostname.split('-')[1]
-        return experiment_id
+        object_id = hostname.split('-')
+
+        if len(object_id) == 1:  #for Noteboooks
```

Comment:
nit: pycharm might have objections against lack of space after #

Reply:
Fixing